js/test/UiHelper.js

```diff
@@ -57,6 +47,8 @@ describe('UiHelper', function () {
     });
 
     describe('reloadHome', function () {
+        // TODO triggers error messages in jsDOM version 11
+        /*
```

Comment:
Similar version thing?

```suggestion
    describe('reloadHome', function () {
        // TODO triggers error messages since jsDOM version 11
        /*
```